python/plugins/processing/algs/grass7/Grass7Utils.py

```diff
@@ -400,6 +400,11 @@ def executeGrass(commands, feedback, outputCommands=None):
         # commands again.
         if not grassOutDone and outputCommands:
             command, grassenv = Grass7Utils.prepareGrassExecution(outputCommands)
+            # For MS-Windows, we need to hide the console window.
+            if isWindows():
+                si = subprocess.STARTUPINFO()
+                si.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+                si.wShowWindow = subprocess.SW_HIDE
             with subprocess.Popen(
```

Comment:
please add also:
encoding="cp{}".format(Grass7Utils.getWindowsCodePage()) if isWindows() else None,
call parameter

Reply:
> I don't feel I will be able to do this...

np, wait only the add encoding parameter, tnx for the fix

Reply:
I've already added the line... or at least I think so... :(

Reply:
If you need anything more from me, please tell me. As I said, I'm not used to github, and I don't know if there is something more that I should have done...